### DISCUSSION TITLE
FNIRSI GC-01: Different readings

### DISCUSSION THREAD

**Discussion by ChyrkaA** [Gissio/radpro#141] [General]
Friends, hello everyone. Another Fnirsi GC-01 radiation detector has just arrived to me. The previous one helped me restore Gissio, but before that I thought that it was damaged forever, so I ordered another one. 

The question is what. Why do devices of the same models display different data (seen in the video). On the left is a new device, based on an APM processor and with an M4011 handset. On the right, my old device, with the CH32F processor. The difference in readings is almost three times. Why is this happening? How to fix this? Which of these devices is lying?

https://github.com/user-attachments/assets/b93f7891-3d43-4e65-acf0-5e4e65ae8504

**Comment by gc01noob** [comment]
some things to consider:
are they the same tubes? did you set the correct profile? there is info in the help files about tube profiles.
what voltage is across the tube? (be very careful with high voltage - read up how to measure before you attempt it, ~20mA can kill you and 400V can easily supply this through skin)
imo the APM is a better machine in every way
my background reading is also approx the same as the one showing 0.222ish - this corresponds to published results online for my area - you can check online for your location
do the readings change as the device warms up?

have you considered purchasing a test sample source to make a more accurate check? these can be purchased from ebay or salvaged from old smoke detectors. or maybe locally you have an area with high background you can use to check? old glassware, clocks and stone can give readings. then see which responds properly.
0.09 seems very low

**Reply by ChyrkaA** [reply]
> some things to consider: are they the same tubes? what voltage is across the tube? (be very careful with high voltage - read up how to measure before you attempt it, ~20mA can kill you and 400V can easily supply this through skin) imo the APM is a better machine in every way my background reading is also approx the same as the one showing 0.222ish - this corresponds to published results online for my area - you can check online for yoru location do the readings change as the device warms up?
> 
> have you considered purchasing a test sample source to make a more accurate check? these can be purchased from ebay or salvaged from old smoke detectors. or maybe locally you have an area with high background you can use to check? old glassware and stone can give readings. then see which responds properly. 0.09 seems very low
> 
> my gut says the 0.222 one is the APM and it's the correct but just a guess.

Thanks for the answer. The device does not heat up at all. The radiation background in my region fluctuates between 0.1 and 0.15 microsieverts. The device that shows 0.222 has a CH32F processor and a J321 tube. I did not measure the voltage in the tube) can this be done with a multimeter)?

**Comment by TzOk83** [comment]
There are ways you can measure the tube voltage using a multimeter and a resistor, and they are shown on the forum (Discussions). From what I understood one meter is using M4011 tube, and the other one J321 tube. Are you sure about the tube types? Are they correctly configured in software? Finally, 0.1 +/-50% and 0.2 +/-50% are not that far from each other. One says that the radiation level is between 0.05 and 0.15 uSv/h, while the second says it is between 0.1 and 0.3 uSv/h...

**Reply by ChyrkaA** [reply]
> There are ways you can measure the tube voltage using a multimeter and a resistor, and they are shown on the forum (Discussions). From what I understood one meter is using M4011 tube, and the other one J321 tube. Are you sure about the tube types? Are they correctly configured in software? Finally, 0.1 +/-50% and 0.2 +/-50% are not that far from each other. One says that the radiation level is between 0.05 and 0.15 uSv/h, while the second says it is between 0.1 and 0.3 uSv/h...

Thanks for the answer. Yes, I'm sure about the tubes. In the RadPRO settings, I selected the tube type accordingly. I would like to understand which one is more correct. And then the difference of almost 3 times is a little confusing. Could you send a link to a topic where measuring tube voltage is discussed? I'll be grateful.

**Comment by gc01noob** [comment]
don't forget that the readings of 0.1 and 0.2 uSv are tiny and within the error limits of these devices - if it were 200 and 600 i'd be concerned. it's worth investigating further but don't get hung up on the "3 times the reading" - different tubes can have different baseline readings. find an actual source and you'll see that it's probably not a huge difference.

WARNING:
be very very careful when measuring tube voltages. they can kill you easily.
(you put a very high value resistor in series on one of the probes and use the reading as a factor of the actual voltage. 1GOhm is common - if this sounds like gibberish don't even consider going near 1000V - or you can buy a super expensive >1000V meter and measure directly)
again, 1 slip of your finger can kill you stone dead.

**Reply by ChyrkaA** [reply]
> don't forget that the readings of 0.1 and 0.2 uSv are tiny and within the error limits of these devices - if it were 200 and 600 i'd be concerned. it's worth investigating further but don't get hung up on the "3 times the reading" - different tubes can have different baseline readings. find an actual source and you'll see that it's probably not a huge difference.
> 
> WARNING: be very very careful when measuring tube voltages. they can kill you easily. (you put a very high value resistor in series on one of the probes and use the reading as a factor of the actual voltage. 1GOhm is common - if this sounds like gibberish don't even consider going near 1000V - or you can buy a super expensive >1000V meter and measure directly) again, 1 slip of your finger can kill you stone dead.

It would be better, after all, to find a test source and test the devices on it. So at least I’ll stay alive))) I’ll post the results.

**Reply by ChyrkaA** [reply]
Below I wrote the measurement results

**Comment by gc01noob** [comment]
makes sense.
if it comes back reading 200 and 203 then just ignore it.
if it comes back 200 and 400 then you have a problem.

**Reply by ChyrkaA** [reply]
> makes sense. if it comes back reading 200 and 203 then just ignore it. if it comes back 200 and 400 then you have a problem.

I don't quite understand what you mean?

**Reply by ChyrkaA** [reply]
Below I wrote the measurement results

**Comment by Gissio** [comment]
To compare two tubes, use CPS or CPM units. Also, use the averaging mode and get an average over 30 minutes or more. This will let you do a much more accurate comparison.

If the difference is below 50%, you are fine, as the M4011 and J321 tubes are quite similar.

At first sight, however, the difference seems to be larger, most likely due to a problem with the high voltage generator in the device with the higher reading (background levels of radiation are usually 0.1-0.15 uSv/h). In this case, you need to set up a custom HV profile by following these instructions: https://github.com/Gissio/radpro/blob/main/docs/users.md#hv-profiles

As other users mentioned, be careful when working with high voltage, as it **can be lethal**.

**Reply by ChyrkaA** [reply]
Below I wrote the measurement results

**Comment by ChyrkaA** [comment]
Hello. I took out a test sample of uranium glass. And measured the average value. I set CPM units in the parameters, as Gissio wrote. The measurement took 30 minutes. Here are the results from the devices:
![photo_2024-09-22_17-37-56](https://github.com/user-attachments/assets/ce39adaf-3052-4fd3-9514-bc7e196ec01a)
![photo_2024-09-22_17-37-56 (2)](https://github.com/user-attachments/assets/e17d15c2-e234-492d-a77b-3f3e53b92d12)
in the first photo - a device with a CH32F processor and a J321 handset, in the second photo, an APM processor and an M4011 handset.

Are both devices working fine? As a result, which of the devices can be said to be more accurate?

**Reply by Gissio** [reply]
The results fall within the expected range of variation, so it seems both devices are working properly.

The difference in observed background level can thus be attributed to differences between your M4011 and J321 tubes (ergo, not due to a problem with the high voltage). If this difference bothers you, go to the Internet and search for websites that monitor the background level of radiation near your area. Use their published value to calibrate the background level of your devices using Rad Pro's background level compensation feature: https://github.com/Gissio/radpro/blob/main/docs/users.md#background-compensation

**Reply by ChyrkaA** [reply]
> The results fall within the expected range of variation, so it seems both devices are working properly.
> 
> The difference in observed background level can thus be attributed to differences between your M4011 and J321 tubes (ergo, not due to a problem with the high voltage). If this difference bothers you, go to the Internet and search for websites that monitor the background level of radiation near your area. Use their published value to calibrate the background level of your devices using Rad Pro's background level compensation feature: https://github.com/Gissio/radpro/blob/main/docs/users.md#background-compensation

Thanks for the answer. If both devices work fine, then I can sell one of them. Tell me, which one is better to keep and which one to sell? Let me remind you that one device has a CH32F processor and a J321 tube, and the second has an APM processor and an M4011 tube.

**Reply by Gissio** [reply]
The APM lets you log much more data than the CH32!

**Reply by ChyrkaA** [reply]
> The APM lets you log much more data than the CH32!

And in terms of work, are they, give or take, the same?

**Reply by Gissio** [reply]
Yes.
